### DISCUSSION TITLE
Unable to set keyboard shortcuts

### DISCUSSION THREAD

**Discussion by samfriedmaan** [MonitorControl/MonitorControl#1622] [Q&A]
<img width="842" alt="Screenshot 2024-09-26 at 3 46 19 PM" src="https://github.com/user-attachments/assets/3968af14-0d2a-4c0c-9489-5626b746b726">

I do not have the ability to set keyboard shortcuts. I am on Sequoia 15.0. I have tried quitting and restarting the app. This issue persists on both 4.3 and 4.3.1

<img width="392" alt="Screenshot 2024-09-26 at 3 49 47 PM" src="https://github.com/user-attachments/assets/43afb933-7b12-46af-9c00-81a98328d899">

**Comment by waydabber** [comment]
Yes, this is indeed an issue.

**Comment by waydabber** [comment]
https://github.com/MonitorControl/MonitorControl/issues/1623
